docs/Image.md

```diff
@@ -31,6 +31,8 @@ The `Image` is a VTEX block that allows to **add any image** in the store.
 | ------------- | ---------- | -------------------------------------------------------------------------- | - |
 | `src`         | `string!`  | Specifies the URL of an image                                              | - |
 | `alt`         | `string`   | Specifies an alternate text for an image                                   | - |
+| `inlineWidth` | `number \| string` | Specifies the inline width attribute of an image                   | - |
+| `inlineHeight`| `number \| string` | Specifies the inline height attribute of an image                  | - |
```

Comment:
```suggestion
| `inlineWidth` | `number` / `string` | Specifies the image's inline width attribute  | - |
| `inlineHeight`| `number` / `string` | Specifies the image's inline height attribute | - |
```

Reply:
I don't understand the scenarios in which the height/width attribute should be written as a string. Could you give me an example?

Reply:
@guerreirobeatriz These attributes also accept `percentages` and `auto` values, so the string option will provide some coverage for those situations

![image](https://user-images.githubusercontent.com/55160541/112485980-3de62c80-8d5a-11eb-9134-33fecd95181d.png)